torch/csrc/jit/serialization/export_bytecode.cpp

```diff
@@ -248,7 +247,60 @@ std::unique_ptr<mobile::Code> compileGraphToMobileCode(
   return mobile_code_ptr;
 }
 
-void checkSchema(const FunctionSchema& schema) {
+std::unique_ptr<mobile::Function> convertJitFunctionToMobileFunction(
+    const GraphFunction& function,
+    const CompilationOptions& options) {
+  BackendDebugInfoRecorder debug_handle;
+  std::shared_ptr<mobile::Code> mobileCode = compileGraphToMobileCode(
+      function.name(), function.graph(), options, debug_handle);
+  const auto& schema = function.getSchema();
+  auto mobile_func = std::make_unique<mobile::Function>(
+      function.qualname(), mobileCode, schema);
+
+  return mobile_func;
+}
+
+IValue convertMobileFunctionToCodeTable(
```

Comment:
It will be made a mobile function again during mobile runtime build. mobile build is a separate from desktop build and it doesn't include `export_bytecode.cpp`. It will include a file like `upgrader_mobile.h` generated from desktop build, which includes a map (key: upgrader name, value bytecode)

As for context, desktop build is `setup.py` and mobile build is like `build_ios.sh`

Reply:
Another option is to just create a table of strings and integers as a bytecode. So we can bypass the converting to IValue part.

Reply:
Actually I feel like creating a table of strings and integers are bit ugly and hard to maintain. Because you would have to create a custom structs to represent them and expose them through pybind. (instruction for example is a tuple of (str, int, int)) Would creating IValue create that much overhead given that this bytecode generation only used during build process? @gmagogsfm @cccclai

Reply:
It wouldn't be that much overhead, just that C++ structures would be cleaner (well defined struct vs generic IValue). Maybe let's leave this as a TODO for future cleanup?

Reply:
Maybe leave it as is and swap to flatbuffer when it's ready?